ContactDetailsApi/V2/Gateways/ContactDetailsDynamoDbGateway.cs

```diff
@@ -110,6 +110,7 @@ public async Task<ContactDetails> CreateContact(ContactDetailsEntity contactDeta
             var contactDetails = new Dictionary<Guid, IEnumerable<ContactDetails>>();
             foreach (var targetId in targetIds)
             {
+                _logger.LogInformation("Fetching contact details for target id: {targetId}", targetId.ToString());
```

Comment:
`.toString()` isn't required in the Logger string formatter. Just `targetId` will do.